readthedocs/templates/projects/domain_list.html

```diff
@@ -42,3 +56,15 @@ <h3> {% trans "Add new Domain" %} </h3>
     </p>
   </form>
 {% endblock %}
+
+{% block footerjs %}
```

Comment:
We don't put javascript inline in templates, it's not maintainable. I'm not convinced this solution needs javascript either way though.